src/js/components/search/filters/contractFilters/ContractFilter.jsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * ContractFilter.jsx
+ * Created by Emily Gullo on 6/23/2017
+ */
+
+import React from 'react';
+import * as Icons from 'components/sharedComponents/icons/Icons';
+import * as ContractFieldDefinitions from 'dataMapping/search/contractFields';
+
+import PrimaryCheckboxType from 'components/sharedComponents/checkbox/PrimaryCheckboxType';
+
+const propTypes = {
+    toggleFilter: React.PropTypes.func,
+    contractFilterType: React.PropTypes.string,
+    contractFilterOptions: React.PropTypes.string,
+    contractFilterState: React.PropTypes.string
+};
+
+const defaultShown = 5;
+
+const defaultState = {
+    shown: defaultShown,
+    shownType: 'more'
+};
+
+export default class ContractFilter extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = defaultState;
```

Comment:
It's the way it's handled in the file I used as a guide - I can move the default values down if that's the preference.